src/iopts.c

```diff
@@ -0,0 +1,58 @@
+/* Copyright 2015-2017. The Regents of the University of California.
```

Comment:
This should go into BART - not the viewer. Also please do not use public github pull requests but our internal git.
- The copyright header make no sense.